### PR TITLE
Add sentence annotator bundle/pipeline

### PR DESCRIPTION
**Pull request by mantognini**
Linked to BlueBrain/bluima#3.

### CODE REVIEW

File: config/bundles/bluima.preprocessing_0.1.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
```
Comment (renaud) [inline config/bundles/bluima.preprocessing_0.1.json]: please add "domain" : "bluima"

File: config/bundles/bluima.preprocessing_0.1.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          "model" : [ "ch.epfl.bbp.nlp.res.sentence.GeniaResource" ]
+        }
+      }, {
+        "name" : "TokenAnnotator",
+        "class" : "ch.epfl.bbp.uima.ae.TokenAnnotator",
+        "parameters" : {
+          "modelFile" : [ "TODO" ]
+        }
+      }, {
+        "name" : "PosTagAnnotator",
```
Comment (renaud) [inline config/bundles/bluima.preprocessing_0.1.json]: this one should get removed?
i need to enforce names to be uniques...

File: pom.xml [hunk 3]
```diff
@@ -177,7 +177,6 @@
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpcore</artifactId>
       <version>4.4</version>
-      <scope>test</scope>
```
Comment (renaud) [inline pom.xml]: can you create a test that replicates this bug? or at least send me a stacktrace of the error? thanks

**Comment by renaud** [comment]
Hi Marco,
Again, some small changes, hopefully.
Thanks for you work.
-- Renaud

File: config/bundles/bluima.preprocessing_0.1.json (hunk 1 above)
Comment (mantognini) [inline config/bundles/bluima.preprocessing_0.1.json]: Sure. But should I also rename it to `preprocessing` only? And should I move it to `config/bundles/bluima` too?

File: config/bundles/bluima.preprocessing_0.1.json (hunk 2 above)
Comment (mantognini) [inline config/bundles/bluima.preprocessing_0.1.json]: Probably yes. I'll do it as well on BlueBrain/bluima.

File: pom.xml (hunk 3 above)
Comment (mantognini) [inline pom.xml]: At commit 0cb6fc2a8b818956f8a2b8d1de85abb4f477e186 (currently master), I get [this error](https://gist.github.com/mantognini/a42137016dc927c3da0f) when clicking on the annotate button of the demo.

File: config/bundles/bluima.preprocessing_0.1.json (hunk 1 above)
Comment (renaud) [inline config/bundles/bluima.preprocessing_0.1.json]: no & no, thanks

File: config/bundles/bluima.preprocessing_0.1.json (hunk 2 above)
Comment (renaud) [inline config/bundles/bluima.preprocessing_0.1.json]: thx

File: pom.xml (hunk 3 above)
Comment (renaud) [inline pom.xml]: ok. thanks. let's remove scope test then.

File: config/bundles/bluima.preprocessing_0.1.json (hunk 1 above)
Comment (mantognini) [inline config/bundles/bluima.preprocessing_0.1.json]: implemented

**Comment by mantognini** [comment]
I forgot to mention that this PR had also been updated.

**Pull request merged**